sentry-kotlin-multiplatform/src/commonMain/kotlin/io/sentry/kotlin/multiplatform/Scope.kt

```diff
@@ -0,0 +1,216 @@
+package io.sentry.kotlin.multiplatform
+
+import io.sentry.kotlin.multiplatform.protocol.Breadcrumb
+import io.sentry.kotlin.multiplatform.protocol.User
+
+class Scope constructor(private val scope: ISentryScope) : ISentryScope {
```

Comment:
h: No attachments for KMP scope?

Reply:
I will add more features in the next couple days/weeks, the plan was to introduce some basic Scope options first